service/storage_service.cc

```diff
@@ -497,6 +497,7 @@ class topology_mutation_builder {
     topology_mutation_builder& set_transition_state(topology::transition_state);
     topology_mutation_builder& set_current_cdc_generation_id(const cdc::generation_id_v2&);
     topology_mutation_builder& set_new_cdc_generation_data_uuid(const utils::UUID& value);
+    topology_mutation_builder& set_global_topology_request(std::optional<global_topology_request>);
```

Comment:
having `del` is better then passing an optional and we already do that for others.